### PR TITLE
Adding a filter for tags to run builds for tag releases

### PR DESCRIPTION
**Pull request by moekify**
Currently, tagged releases are not creating workflows on CircleCI.
This change should make sure that we also run the build for created tags and hence deploy production.

Including a fix for the og-title on a subpage to see we have a working deploys :)

### CODE REVIEW

File: src/locales/seo.json [hunk 1]
```diff
@@ -129,6 +129,14 @@
...
       "og-image": "",
       "og-image-alt": "$t(general:company-name) | $t(general:en.tagline)"
+    },
+    "invite": {
+      "title": "Company Invite Request received",
+      "description": "Honeypot received your request",
+      "og-title": "Company Invite Request received",
+      "og-description": "Honeypot received your request",
+      "og-image": "",
+      "og-image-alt": "$t(general:company-name) | $t(general:en.tagline)"
```
Comment (jneurock) [inline src/locales/seo.json:139]: What's up with this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -113,4 +113,9 @@ jobs:
 workflows:
   build-and-deploy:
     jobs:
-      - build
+      - build:
+          filters:
+            tags:
+              # Match v1.0.0 but skip with a suffix e.g. '-auto
+              only: /v[0-9]+(\.[0-9]+)*$/ # v1.0.0
```
Comment (jneurock) [inline .circleci/config.yml:120]: Looks good 🤞

File: src/locales/seo.json (hunk 1 above)
Comment (moekify) [reply src/locales/seo.json:139]: I added a fix for that page title as it is currently broken. This also gives an easy way to verify deploys are actually working as expected :)

Comment (moekify) [reply src/locales/seo.json:139]: e.g. check page title here rn:

staging.honeypot.co/company_invite_received

**Comment by moekify** [comment]
Task linked: [CU-a4xpx6 Migrate from Travis CI](https://app.clickup.com/t/a4xpx6)

File: src/locales/seo.json (hunk 1 above)
Comment (jneurock) [reply src/locales/seo.json:139]: Thanks

**Review by jneurock — APPROVED** [review]

**Pull request merged**